python/Ganga/GPIDev/Lib/Registry/RegistrySlice.py

```diff
@@ -304,7 +301,8 @@ def copy(self, keep_going):
         return this_slice
 
     def __contains__(self, j):
-        return j.id in self.objects.keys()
+        allObjK = self.objects.keys()
+        return j.id in allObjK
```

Comment:
This was to reduce the objects.keys potentially being called multiple times. I'm no expert on when python evaluates commands on inline stuff like this and I was going for clarity that this was called just once.